src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/102.diff.sql

```diff
@@ -0,0 +1,523 @@
+/***************************************************************
+* Migration: 102
+* Description: Add new column DecompressedSize to Resource table to store the size of decompressed resource data.
+
+To avoid impacting existing functionality, the new column is added as nullable.
+To make this change both backward and forward compatible,
+A new type ResourceList_Temp will be created to include this new column.
+MergeResources SP will be changed to use both new and old types in iteration1.
```

Comment:
3 stored procedures should be changed, not just MergeResources.
It is worth mentioning the plan for backfilling DecompressedLength